### ISSUE TITLE
stack-walk test failure

### ISSUE THREAD

**Issue by jryans** [stephenrkell/liballocs#40]
On at least Ubuntu 18.04.2, the test `stalk-walk` fails at build time with:

```
$ make gdbrun-stack-walk
make -C "stack-walk" `if test -e /home/jryans/Projects/liballocs/tests//stack-walk/mk.inc; then /bin/echo -f mk.inc; else true; fi` "stack-walk" && ( cd "stack-walk" && make `if test -e /home/jryans/Projects/liballocs/tests//stack-walk/mk.inc; then /bin/echo -f mk.inc; else true; fi` -f ../Makefile _onlygdbrun-stack-walk )
make[1]: Entering directory '/home/jryans/Projects/liballocs/tests/stack-walk'
/home/jryans/Projects/liballocs/tools/lang/c/bin/allocscc -L\/home/jryans/Projects/liballocs/contrib/libantlr3c-3.4/.libs -Wl,-rpath,\/home/jryans/Projects/liballocs/contrib/libantlr3c-3.4/.libs -L/home/jryans/Projects/liballocs/tests/../lib -L/home/jryans/Projects/liballocs/tests/../src  stack-walk.o   -lallocs -o stack-walk
/home/jryans/Projects/binutils-gdb/build/bin/ld: stack-walk.linked.o: in function `__liballocs_walk_stack':
/home/jryans/Projects/liballocs/tests/../include/liballocs.h:771: undefined reference to `_Ux86_64_getcontext'
/home/jryans/Projects/binutils-gdb/build/bin/ld: /home/jryans/Projects/liballocs/tests/../include/liballocs.h:772: undefined reference to `_Ux86_64_init_local'
/home/jryans/Projects/binutils-gdb/build/bin/ld: /home/jryans/Projects/liballocs/tests/../include/liballocs.h:774: undefined reference to `_Ux86_64_get_reg'
/home/jryans/Projects/binutils-gdb/build/bin/ld: /home/jryans/Projects/liballocs/tests/../include/liballocs.h:778: undefined reference to `_Ux86_64_get_reg'
/home/jryans/Projects/binutils-gdb/build/bin/ld: /home/jryans/Projects/liballocs/tests/../include/liballocs.h:788: undefined reference to `_Ux86_64_get_reg'
/home/jryans/Projects/binutils-gdb/build/bin/ld: /home/jryans/Projects/liballocs/tests/../include/liballocs.h:789: undefined reference to `_Ux86_64_get_reg'
/home/jryans/Projects/binutils-gdb/build/bin/ld: /home/jryans/Projects/liballocs/tests/../include/liballocs.h:791: undefined reference to `_Ux86_64_get_reg'
/home/jryans/Projects/binutils-gdb/build/bin/ld: /home/jryans/Projects/liballocs/tests/../include/liballocs.h:797: undefined reference to `_Ux86_64_step'
/home/jryans/Projects/binutils-gdb/build/bin/ld: /home/jryans/Projects/liballocs/tests/../include/liballocs.h:800: undefined reference to `_Ux86_64_get_reg'
/home/jryans/Projects/binutils-gdb/build/bin/ld: /home/jryans/Projects/liballocs/tests/../include/liballocs.h:801: undefined reference to `_Ux86_64_get_reg'
/home/jryans/Projects/binutils-gdb/build/bin/ld: /home/jryans/Projects/liballocs/tests/../include/liballocs.h:803: undefined reference to `_Ux86_64_get_reg'
collect2: error: ld returned 1 exit status
<builtin>: recipe for target 'stack-walk' failed
```

**Comment by stephenrkell** [comment]
Hmm. Seems to be a consequence of 49afa666 which (among other things) made using the actual libunwind runtime the default, rather than our hacked-up (but super-fast) frame-pointer-only version. I've just pushed a tentative fix in f093313... let me know if that helps.

**Comment by jryans** [comment]
Yes, https://github.com/stephenrkell/liballocs/commit/f093313d0f39b1aba6f9df5407886152035836be seems to fix this, thanks! :smile:
